update config to match new version of dropbox uploader

fdarveau opened the pull request:
Excerpt from [Dropbox Uploader README.md](https://github.com/andreafabrizi/Dropbox-Uploader/blob/master/README.md#upgrading-from-old-dropbox-api) :

>Starting September 30th, 2021, Dropbox is updating their API (OAuth scopes, PKCE, refresh tokens, and short-lived access tokens) dropbox_uploader.sh configurations made with the old API will not longer work after that date. Reconfigure dropbox_uploader.sh.

It seems Dropbox Uploader has already removed support for OAUTH_ACCESS_TOKEN, so it is needed to update this repository as well since the Dockerfile is pulling the latest version of Dropbox Uploader.

We could also just update the Dockerfile to pull a specific version of Dropbox Uploader, but we would need the changes in this PR by September 30th 2021 anyway.

The steps to configure this container after the update are a bit more complicated than previously, so I tried simplifying them using existing web apps that allow sharing base configs so users will only have to replace some variables to get the data they need.

fdarveau committed:
update config to match new version of dropbox uploader and add interactive setup

shivpatel commented:
Yikes this looks like a very involved process. We may want to re-approach this by introducing a new run-mode for the image that can do all this for the user without needing 3rd party websites/resources.

From my understanding, you can run the original uploader script wizard to guide you through the OAuth process.

> The first time you run dropbox_uploader, you'll be guided through a wizard in order to configure access to your Dropbox. This configuration will be stored in ~/.dropbox_uploader.

Source: https://github.com/andreafabrizi/Dropbox-Uploader#getting-started

Is that correct?

fdarveau commented:
I had the same reaction while setting this up.

Using the interactive mode is not ideal either in my opinion, though it is less involved than what I added in this PR.

I created a sample PR in my forked repo : https://github.com/fdarveau/bitwarden_rs_dropbox_backup/pull/1/files. I added a `/config` volume mount so the setup will need to be ran only once, even if the container is recreated from scratch.

If you prefer the interactive setup, I will push it into my master branch to update this PR.

shivpatel commented:
> I had the same reaction while setting this up.
> 
> Using the interactive mode is not ideal either in my opinion, though it is less involved than what I added in this PR.
> 
> I created a sample PR in my forked repo : https://github.com/fdarveau/bitwarden_rs_dropbox_backup/pull/1/files. I added a `/config` volume mount so the setup will need to be ran only once, even if the container is recreated from scratch.
> 
> If you prefer the interactive setup, I will push it into my master branch to update this PR.

Just checked out your interactive setup branch. I'm down to merge that in; should be a better UX for the users.

shivpatel commented:
@fdarveau can you squash the commits into one? I'd do it for you but git won't allow me. 

```
git reset --soft HEAD~2
git add .
git commit -m "msg"
git push -f origin main
```

fdarveau commented:
Done
